src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -79,6 +79,8 @@ enum class AttestationVerificationResult : uint16_t
 
     kInternalError = 900,
 
+    kPaiAndDacRevoked = 1000,
```

Comment:
I'd expect this to be up near the other Pai/Dac bits, with a similar value.  Not sure whether to put it with the kDac or kPai constants; which of those is the more immediate problem if both are revoked?

Reply:
Moved closer to 'kPai` bits, as the name starts with 'kPai` and  revoked PAI is a bigger concern, revoked DAC.